src/api/app/models/workflow.rb

```diff
@@ -29,7 +29,7 @@ def call
       destroy_target_projects
     when scm_webhook.reopened_pull_request?
       restore_target_projects
-    when scm_webhook.new_pull_request?, scm_webhook.updated_pull_request?, scm_webhook.push_event?
+    when scm_webhook.new_pull_request?, scm_webhook.updated_pull_request?, scm_webhook.push_event?, scm_webhook.tag_push_event?
```

Comment:
This change makes the specs pass. However, push events for tags still won't trigger a workflow since `ScmWebhookEventValidator` doesn't accept those events. This is the final step to enable push events for tags.